item.py

```diff
@@ -0,0 +1,7 @@
+class Item:
```

Comment:
I'd be tempted to rename this to something less generic than `Item` (e.g. `TrelloCard`)